hwdb/60-sensor.hwdb

```diff
@@ -184,4 +187,3 @@ sensor:modalias:acpi:BMA250*:dmi:*:bvrTREK.G.WI71C.JGBMRBA*:*:svnInsyde:pnST7041
 #########################################
 sensor:modalias:acpi:*KIOX000A*:dmi:*svn*CytrixTechnology:*pn*Complex11t*
  ACCEL_MOUNT_MATRIX=-1, 0, 0; 0, 1, 0; 0, 0, 1
-
```

Comment:
Can you please remove this line removal from the patch?

Reply:
Hmm, it removes trailing whitespace. I am pretty sure such a trailing line should not be there, hence I figure we should just merge this. Yes, this could be split up in multiple commits or even PRs, but I am not sure this is really worth the effort, and we snuck in whitespace fixes in other areas of the code all the time. Hence I am inclined to just merge this. I hope this makes sense